docs/.docusaurus/client-modules.js

```diff
@@ -0,0 +1,4 @@
+export default [
+  require("infima/dist/css/default/default.css"),
+  require("/home/bradford/Developer/instant/docs/src/css/custom.css"),
```

Comment:
This wont work on other developers computers. I also think we can ignore the `.docusaurus` directory. Lets create a gitignore to exclude these from the repo
https://github.com/jembi/openhim-docs/blob/master/.gitignore

Reply:
fixed https://github.com/openhie/instant/pull/9/commits/5a61fd1160b5db992bd221d8a65b58f4ac1eacc6